cross/pendulum/patches/001-fix-build-for-32bit.patch

```diff
@@ -0,0 +1,90 @@
+--- /pendulum-3.0.0/rust/src/helpers-orig.rs	2023-12-16 17:04:29
++++ /pendulum-3.0.0/rust/src/helpers.rs	2025-01-19 08:51:27
```

Comment:
@mreid-tt you must avoid absolute paths in patch file
and you must remove the top folder (or define PATCH_LEVEL = 1)

```suggestion
--- rust/src/helpers-orig.rs	2023-12-16 17:04:29
+++ rust/src/helpers.rs	2025-01-19 08:51:27
```